service/src/main/java/uk/gov/hmcts/reform/fpl/service/IdentityService.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.springframework.stereotype.Service;
+
+import java.util.UUID;
+
+@Service
+public class IdentityService {
```

Comment:
it is only to help with testing. Currently we dont have option to spy on static UUID.randomUUID and in many places we cannot just use equals to verify that element is what expected. This can be spied/mocked and in a test we can easily check element object (and verify that it got e.g brand new id)
see ManageHearingsServiceTest.shouldAdjournAndReListHearingWithoutDocumentReassignment

maybe the correct option is to use powermock to have option to spy on static members

Reply:
Using static methods is always fishy IMHO, and wrap them whenever possible. If there are cases anyway when we cannot get rid of them (the wrapped implementation for example) there's now support for that since mockito-inline 3.4.4

https://github.com/hmcts/fpl-ccd-configuration/blob/466e403464d4cb8c039b048a57d6a3ed7f81262b/service/src/test/java/uk/gov/hmcts/reform/fpl/model/order/generated/GeneratedOrderTest.java#L34

https://github.com/hmcts/fpl-ccd-configuration/blob/2d2d9d0682c9e03ac2e0caa102ce909fc2684863/service/build.gradle#L246